CHANGELOG.md

```diff
@@ -171,6 +194,42 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   `revset-aliases.'trunk()'` are now escaped if necessary.
   [#5359](https://github.com/jj-vcs/jj/issues/5359)
 
+### Contributors
+
+Thanks to the people who made this release happen!
+
+* Angel Ezquerra (@AngelEzquerra)
+* Antoine Martin (@alarsyo)
+* Anton Bulakh (@necauqua)
+* Austin Seipp (@thoughtpolice)
+* Baltasar Dinis (@bsdinis)
+* Benjamin Tan (@bnjmnt4n)
+* Bryce Berger (@bryceberger)
+* Charlie-83 (@Charlie-83)
+* Christian Stoitner (@cstoitner)
+* Evan Martin (@evmar)
+* George Christou (@gechr)
+* Ilya Grigoriev (@ilyagr)
+* JDSeiler (@JDSeiler)
+* Jakob Hellermann (@jakobhellermann)
+* Jonathan Frere (@MrJohz)
+* Jonathan Gilchrist (@jgilchrist)
+* Josh Steadmon (@steadmon)
+* Martin von Zweigbergk (@martinvonz)
+* Matt Kulukundis (@fowles)
+* Ollivier Robert (@keltia)
+* Philip Metzger (@PhilipMetzger)
+* Philipp Albrecht (@pylbrecht)
+* Robert Jackson (@rwjblue)
+* Samuel Tardieu (@samueltardieu)
+* Scott Taylor (@scott2000)
+* Stephen Jennings (@jennings)
+* Valentin Gatien-Baron (@v-gb)
+* Vincent Ging Ho Yim (@cenviity)
+* Waleed Khan (@arxanas)
+* Yuya Nishihara (@yuja)
+* blinry (@blinry)
```

Comment:
nit: I have sorted names case-insensitively in the past, so I would have put this after Benjamin

Reply:
Done.